adafruit_rockblock.py

```diff
@@ -24,7 +24,14 @@
   https://github.com/adafruit/circuitpython/releases
 
 """
+from __future__ import annotations
 
+try:
+    from typing import Tuple, Union, Optional
+    from board import UART
```

Comment:
I think we'll want the UART from `busio` instead of `board`.  `busio.UART` is the bus class `board.UART()` initializes and returns the instance of UART mapped to pins labeled on the silkscreen.